tools/istio-docker.mk

```diff
@@ -83,7 +84,8 @@ $(foreach FILE,$(DOCKER_FILES_FROM_ISTIO_BIN), \
 docker.proxy_init: BUILD_ARGS=--build-arg BASE_VERSION=${BASE_VERSION}
 docker.proxy_init: pilot/docker/Dockerfile.proxy_init
 docker.proxy_init: $(ISTIO_DOCKER)/istio-iptables.sh
-docker.proxy_init: $(ISTIO_DOCKER)/istio-iptables
+docker.proxy_init: $(ISTIO_DOCKER)/istio-clean-iptables.sh
+docker.proxy_init: $(ISTIO_DOCKER)/istio-iptables $(ISTIO_DOCKER)/istio-clean-iptables
```

Comment:
I don't think we need the clean script in `istio-init`. A pod always should always start with empty tables?!
You also wrote that explicitly in your `istio-start.sh` changes.
So I would undo all your changes to `tools/istio-docker.mk`.

Reply:
Done in e617b4f75819f834f65078a82bbc68083e392ac5.

Reply:
@rlenglet I reverted the previous revert and added a new commit https://github.com/istio/istio/pull/17044/commits/ab5a2306f804d90865591c9349912a45ae34fc92. I propose to preserve the previous behaviour (clean & add iptables) to the non-Kubernetes environments.